server/__init__.py

```diff
@@ -287,8 +287,12 @@ def create_app(nl_root=DEFAULT_NL_ROOT):
 
   # Load place explorer summaries & allowlist of places to show summaries for
   # Used when rendering place pages
-  app.config['PLACE_SUMMARY_ALLOW_LIST'] = place_summaries.get_place_allowlist()
-  app.config['PLACE_EXPLORER_SUMMARIES'] = place_summaries.get_place_summaries()
+  app.config[
+      'PLACE_SUMMARY_ALLOW_LIST'] = [] if cfg.CUSTOM else place_summaries.get_place_allowlist(
+      )
+  app.config[
+      'PLACE_EXPLORER_SUMMARIES'] = {} if cfg.CUSTOM else place_summaries.get_place_summaries(
+      )
```

Comment:
nit: shall we have a regular if else block so the if block can hold or CUSTOM handlings so it's easier to update later.

Reply:
Done.